docker_config/dockerfiles/Dockerfile.default-cpu

```diff
@@ -64,9 +67,9 @@ ENV SCALA_HOME /usr/share/java
 
 ## Python packages
 
-RUN pip2 install -U pip
+RUN python -m pip install --upgrade "pip < 21.0"
```

Comment:
Why this change?

Reply:
Newer pip2 dropped the support for pip2, the installation will fail.

Reply:
Let's add a comment so it's clear about that